lang/java/avro/pom.xml

```diff
@@ -164,6 +164,11 @@
       <artifactId>xz</artifactId>
       <version>${tukaani.version}</version>
     </dependency>
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <optional>true</optional>
```

Comment:
I went ahead and verified using mvn dependency:tree that this does not pull Joda into projects that depend on Avro transitively.